### PR TITLE
feat: add a server events endpoint

### PR DESCRIPTION
**Pull request by CalvinRodo**
add the server events endpoint to the server metrics tasks

### CODE REVIEW

**Review by mohamed-cds — APPROVED** [review]
🚀 ✅

File: aws/ecs/server_metrics_etl.tf [hunk 1]
```diff
@@ -26,6 +26,7 @@ module "masked_server_metrics_etl" {
     environment             = var.env
     bucket_name             = var.masked_metrics_bucket
     metric_token_secret_arn = aws_secretsmanager_secret.metrics_token.id
+    server_events_endpoint  = "https://retrieval.wild-samphire.cdssandbox.xyz/events"
```
Comment (patheard) [inline aws/ecs/server_metrics_etl.tf:29]: Could this be a variable to make it easier to share between staging/prod?

File: aws/ecs/server_metrics_etl.tf [hunk 2]
```diff
@@ -53,6 +54,7 @@ module "unmasked_server_metrics_etl" {
     environment             = var.env
     bucket_name             = var.unmasked_metrics_bucket
     metric_token_secret_arn = aws_secretsmanager_secret.metrics_token.id
+    server_events_endpoint  = "https://retrieval.wild-samphire.cdssandbox.xyz/events"
```
Comment (patheard) [inline aws/ecs/server_metrics_etl.tf:57]: Same comment about it being a var.

**Review by patheard — CHANGES_REQUESTED** [review]
Question about using a variable.

File: aws/ecs/server_metrics_etl.tf (hunk 1 above)
Comment (CalvinRodo) [reply aws/ecs/server_metrics_etl.tf:29]: Yeah it probably should be ... I did mean to do that

**Review by patheard — APPROVED** [review]
`+1`

**Pull request merged**